src/Http/CurlClient.php

```diff
@@ -15,7 +17,7 @@ class CurlClient implements ClientInterface
     public static function request(
         string $method,
         string $url,
-        array $headers = [],
+        array  $headers = [],
```

Comment:
nitpick: this looks like the indentation problem we had, so one space to much to align variables, not sure how we can solve this permanently, maybe .editorconfig?

Reply:
This is weird. I don't see this space in Github when I click "Edit File". Is Github mistaken?

Reply:
Strange 😂  lets move on